models/Post.js

```diff
@@ -0,0 +1,21 @@
+// This is a code snippet, not used for production.
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const postSchema = new Schema({
+  title: String,
+  postedBy: {
+      type: mongoose.Schema.Types.ObjectId,
```

Comment:
http://mongoosejs.com/docs/populate.html

I think we want to use ObjectId's for all linking.  @Lewellync any input on this?